src/services/public_http_server/handlers/relay_webhook/mod.rs

```diff
@@ -177,7 +177,7 @@ pub struct RelayIncomingMessage {
     pub tag: u32,
 }
 
-#[instrument(skip_all, fields(topic = %msg.topic, tag = %msg.tag, message_id = %get_message_id(&msg.message)))]
+#[instrument(skip_all, fields(message_id = %get_message_id(&msg.message)))]
```

Comment:
I don't think reducing fields should affect the CloudWatch cost here, but let's remove it if you think you don't need it.

Reply:
Why not? CloudWatch charges per GB and more fields is more bytes?

Reply:
@chris13524 is right, AWS does charge by data points, see https://aws.amazon.com/cloudwatch/pricing/